Clients/src/application/authentication/authSlice.ts

```
@@ -132,12 +133,15 @@ const authSlice = createSlice({
 			state.isLoading = false;
 			state.success = true;
 			state.message = "Logged out successfully";
-			localStorage.removeItem("authToken");
+			state.expirationDate = null;
 		},
 		setAuthToken: (state, action: PayloadAction<string>) => {
 			debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)